Fix issues

aubcel opened the pull request:
* fix code issues

aubcel commented on client/src/lib/api.js:
```diff
@@ -1,7 +1,7 @@
 import { possiblePollStates } from "./poll";
 
-const url = "http://127.0.0.1:7999";
+const url = `http://${window.location.hostname}:7999`;
```
I am not sue if that is the solution you wanted?
If not please explain me what you meant 🙂

acidicX commented on client/src/lib/api.js:
```diff
@@ -1,13 +1,18 @@
 import { possiblePollStates } from "./poll";
 
-const url = "http://127.0.0.1:7999";
+const axiosInstance = axios.create({
```
It's probably best to extract this into a lib (`lib/axios.js` or smth), so that you can import it in every file you need axios with the correct baseURL.

acidicX commented on client/vue.config.js:
```diff
@@ -6,5 +6,8 @@ module.exports = {
     }
   },
-  publicPath: "/client/"
+  publicPath: "/client/",
```
maybe this should be documented in the readme, that in dev mode the request is proxied by vue CLI to the backend

aubcel replied:
Yes, we have a issue for the whole readme. It will be updated quite soon.

acidicX commented:
I'm unsure about the `.dockerignore` move to server dir, as you also have node_modules and stuff in the root and that is the context during build. Should probably stay in the root?

aubcel commented:
> I'm unsure about the `.dockerignore` move to server dir, as you also have node_modules and stuff in the root and that is the context during build. Should probably stay in the root?

The top level node_modules shouldn't exist i think. I removed it an it didn't show up again.
But I moved the .gitignore to the toplevel anyways 👍

acidicX commented:
nais work!
